### ISSUE TITLE
Download separate levels

### ISSUE THREAD

**Issue by pagginelli** [wilddom/memrise2anki-extension#82]
Hello. I would like to know if there is some way of downloading levels separately from memrise.

The reason:
I'm using memrise to train French writing and I intend to download it to train speaking on Anki. However, my cards on memrise were automaticaly translated to English using google translator and they have a lot of mistakes that I'm fixing while I study on memrise. Consequently, it would be a lot faster if I could just download the cards that are already corrected in memrise instead of correcting them twice.

Thanks in advance.

**Comment by wilddom** [comment]
No, that's not possible at the moment. You can always update the deck from memrise without losing your progress. But it's downloading the whole deck which may take some time unfortunately.

**Comment by wilddom** [comment]
While technically possible back then, it's not possible with the new API. Downloading the whole deck is required in order to prepare the data correctly.
